contrib/ansible/cri-containerd.yaml

```diff
@@ -26,17 +31,13 @@
         name: net.ipv4.ip_forward
         value: 1
 
-    - name: "Check kubelet args in kubelet config"
-      shell: grep "^Environment=\"KUBELET_EXTRA_ARGS=" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf 
-      ignore_errors: true
-      register: check_args
-
-    - name: "Add runtime args in kubelet conf"
-      lineinfile:
-        dest: "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
-        line: "Environment=\"KUBELET_EXTRA_ARGS= --container-runtime=remote --runtime-request-timeout=15m --image-service-endpoint=/var/run/cri-containerd.sock --container-runtime-endpoint=/var/run/cri-containerd.sock\""
-        insertafter: '\[Service\]'
-      when: check_args.stdout == ""
+    - name: "cri-containerd systemd dropin for kubeadm"
```

Comment:
personally not a fan of this. Would prefer a single config file for the system unit. Especially if ppl intend to modify KUBELET_EXTRA_ARGS and they don't see it in one config file. They might just add it in that. This will cause more issues than help us.

Reply:
It is not an error :) it is an ignored error. It checks if the regex is present / not . If its present it doesn't add. If it is not then it will add.

Reply:
Let's discuss this tomorrow. :)